types/matter-js/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for Matter.js - 0.10.0
+// Type definitions for Matter.js - 0.10.1
```

Comment:
Looks like our lint rules are disabled for this package. Can you enable the lints in our `tslint.json` and fix up the version string to only be a `MAJOR.MINOR` version (e.g. `4.7`), as opposed to including the patch or containing a leading `v` (e.g. `v1.2.3`).

Reply:
@DanielRosenwasser IMO it's work for another pull request. `matter-js/index.d.ts` has a lot of lint errors (especially jsdoc asterisks alignments lol) :D and there's no fixer (nor prettier support) for them.

https://github.com/hasparus/DefinitelyTyped/pull/1

Reply:
Sorry for the delay; just as a heads up, this probably doesn't publish the version as you'd expect.

Reply:
I'll merge this in just not to hold you up too much longer :D